apps/web/components/Menu/Menu.tsx

```diff
@@ -40,7 +41,10 @@ export const Menu = ({
   return (
     <MenuUI
       baseId="Menu"
-      mainLinks={mainLinks}
+      mainLinks={mainLinks.map((item) => ({
+        ...item,
+        dataTestId: 'mega-menu-link',
+      }))}
```

Comment:
`dataTestId` will be non-unique. Maybe use `` `data-testid='mega-menu-link-${item.id}` `` or something.

Also, `dataTestId` should be `data-testid`